guidance/new-projects.md

```diff
@@ -0,0 +1,68 @@
+# New Project Checklist
+
+
+When projects join the foundation, there are two ways that they can come in.
+
+ - One is by the owners of the project signing a contribution agreement licensing
+   the code to the .NET Foundation under the terms of the open source license and
+   confirming that they have the rights to do that. This type of agreement is most
+   common when projects have had a long history of being open source but wish to
+   now be supported by the .NET Foundation.
+   
+ - The other is to assign the copyright of the project to the .NET Foundation and
+   confirm that they have the rights to do this. This type of agreement is usually
+   used when releasing something as open source for the first time or by a project
+   that has had a high degree of control over contributions in the past.  
+
+When you contact the .NET Foundation, they will help you decide which type of
```

Comment:
You might add "based on your needs" to make it clear that the Foundation isn't pushing an agenda, but purely consulting based on the desired outcomes you describe.